ui/src/app/pages/admin/admin-workspace.tsx

```diff
@@ -167,21 +166,20 @@ const NameCell = (props: NameCellProps) => {
   );
 
   // remove first check after RW-5626
-  const isNotebook = () =>
+  const isNotebook =
     NOTEBOOKS_DIRECTORY === parseLocation(file, bucket) &&
     filename.endsWith(NOTEBOOKS_SUFFIX);
-  const isTooLargeNotebook = () =>
-    isNotebook() && file.sizeInBytes > MAX_NOTEBOOK_READ_SIZE_BYTES;
+  const isTooLargeNotebook =
+    isNotebook && file.sizeInBytes > MAX_NOTEBOOK_READ_SIZE_BYTES;
 
   // if (tooLarge()) fileTooLarge();
   // else if (isNotebook()) fileWithPreviewButton();
   // else filenameSpan();
-  const requiredDummyParameter = undefined;
```

Comment:
My understanding as to what happened here:

originally the type signatures were incorrect for this use of `fp.cond()` so a dummy parameter was required to compile.  A new version corrects this, which causes this code to no longer compile.

We can make this moot however, as we have our own `cond()` which is appropriate when there's no argument needed.